examples/howto/using_password_input.py

```diff
@@ -0,0 +1,32 @@
+from bokeh.models.widgets import PasswordInput, TextInput, PreText, Button
+from bokeh.layouts import column, row
+from bokeh.plotting import show, output_file
+from bokeh.models.callbacks import CustomJS
+
+# Set User and Password:
+USER = "Mau"
+PASSWD = "Bok3h"
+
+## Create Texts, inputs and button
+text = PreText(text="LOGIN TO KNOW\nTHE SECRET:")
+user = TextInput(placeholder="username", title="(UserName: "+USER+")")
+pwd = PasswordInput(placeholder="password", title="(Password: "+PASSWD+")")
+btn = Button(label="GO!",width=150)
+
+secret = PreText() # Secret information displayed if correct password entered
+
+## Verify if the password typed is bokeh using a JS script
+verify_pwd = CustomJS(args=dict(user=user, pwd=pwd, secret=secret),
+                      code=""" if (user.value = '{}') & (pwd.value == '{}') {{
+                                   secret.text = 'Right Password. The Secret is 42.';
+                                   return;
+                               }}
+                               secret.text = 'Wrong Password.';
+                               """.format(USER, PASSWD))
+#user.callback = verify_pwd # Check password pressing enter. -BUG Works also with tab :(
+pwd.callback = verify_pwd # Check password pressing enter.  -BUG Works also with tab :(
+btn.callback = verify_pwd # Check password clicking on the Button.
+
+output_file("using_password_input.html", title="Password Field")
+page = row(column(text,user, pwd,btn),secret)
+show(page)
```

Comment:
Agreed - For anything which suggests it could/should be used for security purposes, it's important that we include documentation in both the example, as well as the docstring for the class.  It's not merely to disclaim liability, but also to clearly lay out what aspects of security are *not* addressed by this widget: various ways the password text may be stored in plaintext, various potential attack vectors, SSL/non-HTTPS connections, etc. etc.

I realize this is not a trivial amount of work, but it also doesn't have to be super extensive.. It just needs to give the user the impression that this is not "automagically secure".